app/src/main/java/com/github/polypoly/app/models/game/GameViewModel.kt

```diff
@@ -49,6 +71,62 @@ class GameViewModel(
         gameEndedData.value = gameData.value?.isGameFinished() ?: false
     }
 
+    /**
+     * Close a trade request
+     * @param trade The trade request to close
+     */
+    fun closeTradeRequest(trade: TradeRequest) {
+        tradeRequestData.value = null
+        remoteDB.removeValue<TradeRequest>(trade.code)
+    }
+
+    /**
+     * Update a trade request
+     * @param trade The trade request to update
+     */
+    fun updateTradeRequest(trade: TradeRequest) {
```

Comment:
Such function where you can immediately set the new value generally doesn't lead to a good encapsulation as it is more difficult to ensure the consistency of the value across time: here for example what if the new trade is different then the current one?
I think the best way to improve that would be to only allow modification of the tradeRequest through specific endpoints such as the existing create/close but also by adding new ones such as `accept` (see below the related comment) and something like `addCompensation` for when the receiver adds his location in the trade